### ISSUE TITLE
Issue in CommandLine.cpp

### ISSUE THREAD

**Issue by cybik** [LukasBanana/XShaderCompiler#23]
Compiling on Gentoo Linux.

```
[ 89%] Built target xsc_core
[ 90%] Building CXX object CMakeFiles/xsc.dir/src/Shell/CommandLine.cpp.o
/home/cybik/svnrepos/indivisible-full/trunk/tools/3rdparty/XShaderCompiler/src/Shell/CommandLine.cpp: Dans la fonction membre ‘bool Xsc::Util::CommandLine::AcceptBoolean(bool)’:
/home/cybik/svnrepos/indivisible-full/trunk/tools/3rdparty/XShaderCompiler/src/Shell/CommandLine.cpp:92:69: erreur : no matching function for call to ‘transform(std::__cxx11::basic_string<char>::iterator, std::__cxx11::basic_string<char>::iterator, std::__cxx11::basic_string<char>::iterator, <unresolved overloaded function type>)’
     std::transform(arg.begin(), arg.end(), arg.begin(), std::toupper);

```

**Comment by cybik** [comment]
Per http://en.cppreference.com/w/cpp/algorithm/transform, here's my fix

```
bool CommandLine::AcceptBoolean(bool defaultValue)
{
    auto arg = Get();
    std::transform(arg.begin(), arg.end(), arg.begin(),
                   [](unsigned char c) { return std::toupper(c); });
```

**Comment by LukasBanana** [comment]
Fixed with [1afe52b](https://github.com/LukasBanana/XShaderCompiler/commit/1afe52b0a927f2d45b2b5753ab2ef3c3205034ca)
